### ISSUE TITLE
villagers keep desapawning  

### ISSUE THREAD

**Issue by thisisnotcool4345** [Luke100000/minecraft-comes-alive#266]
so when i left to a different village via way stone from the way stone mod i come back and im unable to locate my wife or the cultist completely disappeared they did not die as no message pop up nor where there names on the grave stones i checked my blue print and the two of them where still there but when i went and used the whistle to call her to me it dosnt even show up that i have any family member in the area and the problem is no they didn't die to a mob as im playing in peaceful only and it looks like they just vanished into the void some where this has happened many times before with the cultist person randomly disappearing but the spouse thats the first 

second problem im un able to procreate with spouse at all and the hearts are at 300 no option is showing up  and no villagers are assign to any task

**Comment by alexust98** [comment]
1. You could try using Whistle item to teleport family members back to you
2. You can try command /tp @e[name="#NPC_name"] @s
3. Use NBTExplorer to find coordinates of target entity

**Comment by SDPhantom** [comment]
From previous experience, the whistle doesn't recognize family members that are in other dimensions and I've had a few that wandered into a nether portal blocked by doors.

**Comment by IndolentKheper** [comment]
Had the same problem with a villager, a cultist I had trapped in a small building so I could have a zombie infect and cure her (which then allows her to stop being a cultist and take a job). Playing in 1.18 Fabric. There isn't a command to TP villagers, is there? No evidence of her death and still listed on the blueprint. Still taking up a bed at the Inn.

**Comment by Wilevar** [comment]
I'm having a similar issue in the current 1.19 release.
My blueprint shows 7 of 18 villagers, but only 3 are present in town. Several of them have just vanished, and I can't teleport the missing characters. Despite that, they still seem to "function" as existing villagers. A pair of them just had a baby, but the mother is no where to be seen. o_o

**EDIT:** I figured out that telling Adventurers to set the Inn as their home through interaction added them to the villager population, and then they despawned some time later. Sometimes after getting married to a villager before hand. I ended up doing this because they wouldn't stay in the Inn at night.

Removing the Inn building through the blueprint cleared them from the population, so that was at least fix-able.
